policies/attribute_name_collisions.rego

```diff
@@ -1,5 +1,6 @@
-package after_resolution
+package attribute_name_collisions
```

Comment:
I’m not quite sure I understand the direction we’re following here. If this policy file is no longer imported directly or indirectly into a package named by one of the stages supported by Weaver, then these policies will no longer be used by Weaver. Consequently, these policies won’t be able to access any OTel-oriented/custom Rego functions or extensions that Weaver could introduce in the future (for perf or ease of use purposes). Furthermore, Weaver is based on Regorus, which is a faster implementation of OPA/Rego, so I’m not sure I understand the reasoning. If the idea is to use Rego to test our policies, we can achieve that by using Regorus itself.

Reply:
no intention, just me misunderstanding what package means